c8run/endpoints.txt

```diff
@@ -1,13 +1,17 @@
 -------------------------------------------
-Access each component at the following urls:
+Access each component at the following urls with these credentials:
+- username: demo
+- password: demo
```

Comment:
Username and password options can be changed with --username and --password. Do you want these options to affect what is outputted by this endpoints.txt file?

Reply:
Good point - I added "default" to the message.